crates/next-api/src/versioned_content_map.rs

```diff
@@ -84,10 +84,7 @@ impl VersionedContentMap {
         let assets = *assets_operation.await?;
         this.map_op_to_compute_entry
             .update_conditionally(|map| map.insert(assets, compute_entry) != Some(compute_entry));
-        let Some(entry) = &*compute_entry.await? else {
-            unreachable!("compute_entry always returns Some(MapEntry)")
-        };
```

Comment:
Can we just change the signature of the `compute_entry` function?

Reply:
Not sure what you mean. It's intentionally an OptionMapEntry, so it can be returned by raw_get directly without additional wrapping.